### DISCUSSION TITLE
When I tried to use the Latest Version of Tailwinds I get Syntax Error: Error: PostCSS plugin tailwindcss requires PostCSS 8.

### DISCUSSION THREAD

**Discussion by udemezue01** [tailwindlabs/tailwindcss#7824] [Help]
Syntax Error: Error: PostCSS plugin tailwindcss requires PostCSS 8.
Migration guide for end-users:
https://github.com/postcss/postcss/wiki/PostCSS-8-for-end-users

 @ ./src/assets/css/tailwinds.css 4:14-173 15:3-20:5 16:22-181
 @ ./src/main.js
 @ multi (webpack)-dev-server/client?http://192.168.92.247:8080&sockPath=/sockjs-node (webpack)/hot/dev-server.js ./src/main.js

I am using Vue JS to do this.

**Comment by AviAvinav** [comment]
I don't use vue js myself but I have seen that error before. In the v2 docs, it was written how to resolve this issue, it's not there in the v3 docs though. 

This happens because many tools have still not updated to postcss 8.

The solution to this problem is to first uninstall tailwindcss and then reinstall it like this:

```bash
npm uninstall tailwindcss postcss autoprefixer      # or yarn

npm install -D tailwindcss@npm:@tailwindcss/postcss7-compat postcss@^7 autoprefixer@^9       # or yarn
```

To know more about this issue, you can check out: https://v2.tailwindcss.com/docs/installation#post-css-7-compatibility-build

Hope it helps.

**Comment by udemezue01** [comment]
I know about this solution, but does this means that we would not be able
to make use of the recent version of tailwinds CSS

**Reply by AviAvinav** [reply]
I think this postcss issue is resolved now, try reinstalling the latest versions of tailwindcss, postcss, autoprefixer. And update the config accordingly. Should probably work. If it doesn't work, please share your tailwind config and dependencies in your package.json.
